modules/swagger-codegen/src/main/resources/nancyfx/Project.mustache

```diff
@@ -39,8 +39,8 @@
         <DocumentationFile>bin\Release\{{packageName}}.XML</DocumentationFile>
     </PropertyGroup>
     <ItemGroup>
-        <Reference Include="Nancy, Version=1.4.1.0, Culture=neutral, processorArchitecture=MSIL">
-            <HintPath>..\..\packages\Nancy.1.4.1\lib\net40\Nancy.dll</HintPath>
+        <Reference Include="Nancy, Version=1.4.2.0, Culture=neutral, processorArchitecture=MSIL">
```

Comment:
That is weird. Nuget does that sometimes, I just wanted to check that it wasn't a typo. I think this version is the version from nuspec while the following line is from assembly info.